src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -526,4 +526,61 @@ protected function getErrorMessageForException(Exception $e, array $messages)
             $e->getMessage()
         );
     }
+
+    /**
+     * Adds a new stylesheet(s) to the page header.
+     *
+     * @param string|array $cssUri Path to CSS file, or list of css files like this : array(array(uri => media_type), ...)
+     * @param string $cssMediaType
+     * @param int|null $offset
+     * @param bool $checkPath
+     */
+    protected function addCSS(
+        $cssUri,
+        $cssMediaType = 'all',
+        $offset = null,
+        $checkPath = true): void
+    {
+        $this->getContext()->controller->addCSS(
```

Comment:
No need to check if `controller` is present here? If I remember correctly a legacy controller is always created along with the context, but it's a behaviour we aim to remove in time, so it might no always be the case
I also remember of cases when this `controller` field was not properly set (CLI or Installer environment I'm not sure any more) so even as it is now maybe it's better to check it

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/19277/commits/db3bcbb7490699d6071d2094d1ab1f708bb3220c